src/backends/interflop-mca/interflop_mca.c

```diff
@@ -408,6 +439,32 @@ error_t parse_opt(int key, char *arg, struct argp_state *state) {
                    key_mode_str);
     }
     break;
+  case KEY_ERR_MODE:
+    /* mca error mode */
+    if (strcasecmp(MCA_ERR_MODE_STR[mca_err_mode_rel], arg) == 0) {
+      ctx->relErr = true;
+      ctx->absErr = false;
+    } else if (strcasecmp(MCA_ERR_MODE_STR[mca_err_mode_abs], arg) == 0) {
+      ctx->relErr = false;
+      ctx->absErr = true;
+    } else if (strcasecmp(MCA_ERR_MODE_STR[mca_err_mode_all], arg) == 0) {
```

Comment:
I'm not sure when I would use the "all" mode. Maybe we can simplify this and include only either absolute or relative but never both.